tests/integration/targets/copr/tasks/main.yml

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
You need to add a file tests/integration/targets/copr/aliases, otherwise the tests are ignored in CI. See other aliases files for examples of what can/should be in there.